dali/base/dafdesc.cpp

```diff
@@ -3701,9 +3701,20 @@ class CStoragePlaneInfo : public CInterfaceOf<IStoragePlane>
     {
         return xml->queryProp("storageapi/@account");
     }
-    virtual const char * queryStorageContainer() const override
+    virtual const char * queryStorageContainer(unsigned stripeNumber) const override
     {
-        return xml->queryProp("storageapi/@container");
+        StringBuffer path("storageapi/");
+        if (stripeNumber==0)
```

Comment:
question: it doesn't feel right quite right to me, that there's 1 specification for listing the storage container if no striping ("@container") and a different way if striping is on (list under "containers").

We pobably need to support the old way (@container) without striping for backward compatibility,
but going forward, I think you should be able to specify the single container under the "containers" list, in the case of a plane witout striping, so it's consistent.
@ghalliday - what's your opinion on this?

Reply:
That is a reasonable idea, but that can be a follow on change - we need to get this merged into 8.10.x asap.

Reply:
ok. @shamser please open a JIRA for this change.

Reply:
I see you've changed it.

It does look cleaner with change, but backward compatibility has been lost.
@shamser @ghalliday - Maybe that isn't important, if no one is actively using it so far?